src/pages/RFOX/components/Stake/hooks/useRfoxStake.tsx

```diff
@@ -125,11 +124,8 @@ export const useRfoxStake = ({
   }, [amountCryptoBaseUnit, isValidStakingAmount, runeAddress, stakingAsset])
 
   const pubKey = useMemo(
-    () =>
-      wallet && isLedger(wallet) && stakingAssetAccountId
-        ? fromAccountId(stakingAssetAccountId).account
-        : undefined,
-    [stakingAssetAccountId, wallet],
+    () => (stakingAssetAccountId ? fromAccountId(stakingAssetAccountId).account : undefined),
```

Comment:
remove in favor of existing `stakingAssetAccountAddress`

Reply:
7e2afce302